gremlin-go/dsl/go.mod

```diff
@@ -0,0 +1,31 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Note: this will likely mess up releases if it's committed.

Reply:
Ack, and yes this whole DSL package will be out of the actual commit.